terraform/cloudfront.tf

```
@@ -2,7 +2,7 @@ resource "aws_cloudfront_distribution" "data_api" {
   enabled             = true
   is_ipv6_enabled     = true
   price_class         = "PriceClass_All"
-  aliases             = [replace(var.service_url, "https://", "")]
+  aliases             = [var.environment == "dev"? "${local.project}-data-api.globalforestwatch.org" : replace(var.service_url, "https://", "")]
```

Hmm, I don;t think this will work. We don't point any `globalforestwatch.org` domain to api services in the dev account. Can you leave the aliases blank?
```suggestion
  aliases             = [var.environment == "dev" ? null : replace(var.service_url, "https://", "")]
```

---

You can't leave it blank unfortunately, terraform will throw an error, so that's why I just added a dummy CNAME. You can't actually go the URL, you just have to go the CloudFront distribution URI. But it's ok with having an unreachable CNAME.

---

terraform docs say it is optional, so null should normally work. Odd. Ok, what ever works, but please leave a comment in the code on why you picked this name for later reference